cobc/scanner.l

```diff
@@ -776,6 +776,46 @@ H#[0-9A-Za-z]+ {
 	RETURN_TOK (DATA);
 }
 
+"WITH"[ ,;\n]+"ASA"/[ .,;\n] {
```

Comment:
Did you get compiler conflicts you could not resolve when adding WITH in front of the options in `with_clause`? Otherwise I wouldn't understand the use of the scanner change.